rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -1687,35 +1716,41 @@ error_code AudioSetNotifyEventQueue(u64 key, u32 iFlags)
 	}
 
 	// Set unique source associated with the key
-	g_audio.keys.push_back({
+	g_audio.keys.push_back
```

Comment:
What is this random change? I'm pretty sure we never did that.

Reply:
We do, search for `cmd_list` function as an example.